pkg/asset/installconfig/azure/validation_test.go

```diff
@@ -50,6 +56,12 @@ var (
 	invalidateRegionLetterCase   = func(ic *types.InstallConfig) { ic.Azure.Region = "Central US" }
 	removeVirtualNetwork         = func(ic *types.InstallConfig) { ic.Azure.VirtualNetwork = "" }
 	removeSubnets                = func(ic *types.InstallConfig) { ic.Azure.ComputeSubnet, ic.Azure.ControlPlaneSubnet = "", "" }
+	invalidateDiskType           = func(ic *types.InstallConfig) {
+		ic.Azure.DefaultMachinePlatform.OSDisk.DiskType, ic.Azure.Region = invalidResourceSkuDiskType, validRegion
```

Comment:
`validInstallConfig` should already have `validRegion` so you shouldn't need to do this. Would be nice to keep formatting consistent and on one line.